assets/js/modules/thank-with-google/components/setup/SetupCreatePublication.js

```diff
@@ -48,6 +60,7 @@ export default function SetupCreatePublication() {
 		>
 			<Button
 				href={ createPublicationURL }
+				onClick={ handleCreatePublication }
```

Comment:
While this may work in some cases, we want to ensure the event is tracked successfully (or times out) before navigating.

Reply:
See my other comment.

Reply:
I think we should add code comments to these just so no one else thinks the same thing—I did when reviewing this code as well 😅